src/libraries/System.Runtime/src/System.Runtime.csproj

```diff
@@ -5,6 +5,8 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <EnablePreviewFeatures>True</EnablePreviewFeatures>
```

Comment:
@jeffhandley: This is the only significant change since you approved the PR.

Reply:
Thanks for the pointer. It's probably worth adding a comment above these two lines to call out how they interact with one another to produce the result we want. It would be easy for us to accidentally remove the `<GenerateRequiresPreviewFeaturesAttribute>False</>` line and "break the world." 🙂